### PR TITLE
change description to policy leaf

### PR DESCRIPTION
**Pull request by MaratMussabekov**
issue #4558

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/client-management/mdm/applocker-csp.md [hunk 1]
```diff
@@ -186,6 +172,8 @@ certutil -encode WinSiPolicy.p7b WinSiPolicy.cer
 </tbody>
 </table>
 
+> [!NOTE]
+> To use Code Integrity Policy, you need first to convert the policies to binary format using the ConvertFrom-CIPolicy cmdlet. Then a Base64-encoded blob of the binary policy representation should be created (for example, using [certutil -encode](https://go.microsoft.com/fwlink/p/?LinkId=724364) command line tool) and added to the Applocker-CSP
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/applocker-csp.md]:
```suggestion
> To use Code Integrity Policy, you first need to convert the policies to binary format using the ConvertFrom-CIPolicy cmdlet. Then a Base64-encoded blob of the binary policy representation should be created (for example, using the [certutil -encode](https://go.microsoft.com/fwlink/p/?LinkId=724364) command line tool) and added to the Applocker-CSP.
```

File: windows/client-management/mdm/applocker-csp.md [hunk 2]
```diff
@@ -156,22 +156,8 @@ Each of the previous nodes contains one or more of the following leaf nodes:
...
-
-<p>An alternative to using certutil would be to use the following PowerShell invocation:</p>
-
-```
-[Convert]::ToBase64String($(Get-Content -Encoding Byte -ReadCount 0 -Path <bin file>))
-```
-
-<p>If you are using hybrid MDM management with System Center Configuration Manager or using Intune, ensure that you are using Base64 as the Data type when using Custom OMA-URI functionality to apply the Code Integrity policy.</p>
-<p>Data type is string. Supported operations are Get, Add, Delete, and Replace.</p></td>
+<p>For nodes, other than CodeIntegrity, policy leaf data type is string. Supported operations are Get, Add, Delete, and Replace.</P>
```
Comment (illfated) [inline windows/client-management/mdm/applocker-csp.md]: Cosmetic : lowercase characters are preferred in tags, unless specified as uppercase in HTML documentation.
```suggestion
<p>For nodes, other than CodeIntegrity, policy leaf data type is string. Supported operations are Get, Add, Delete, and Replace.</p>
```

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9, @illfated ,
you recommendations were applied,
Thank you!

**Comment by mypil** [comment]
@lomayor - PR has been copy edited and is ready for final review. This is based on issue #4558.

Thank you.

**Comment by illfated** [comment]
@Air-Git : maybe you would like to have a look at the resulting change and say what you think.

The rich text diff result can be viewed in this page view: https://github.com/MicrosoftDocs/windows-itpro-docs/pull/4585/files?short_path=969954f#diff-969954f5b69183a668556046006eb41b

**Comment by Air-Git** [comment]
The section under "Policy" is now correct (specifying the data types). However the Note below (regarding certutil) is not how it actually works in Intune.
The _actual_ way to implement the policy is as I wrote previously:
- Encode the XML as a binary file, using ConvertFrom-CIPolicy
- Configure the OMA-URI setting as a Base64 data type
- Upload the binary file (not Base64).
It could be that the documentation is incorrect; or it could be that Intune has not implemented it correctly.

**Comment by illfated** [comment]
@Air-Git : Thank you for your valuable feedback.

@MaratMussabekov : Any thoughts on the comment above?

**Comment by MaratMussabekov** [comment]
Dear @Air-Git ,
do you think if change Note to the following would be more correct? (I took the section about OMA-URI from the original version of the article) 
> [!NOTE]
> To use Code Integrity Policy, you need to convert the policies to binary format using the ConvertFrom-CIPolicy cmdlet. 
> If you are using hybrid MDM management with System Center Configuration Manager or using Intune, ensure that you are using Base64 as the Data type when using Custom OMA-URI functionality, then upload the binary file as a value.

Thank you

**Comment by Air-Git** [comment]
AFAIK, SCCM and GPO provide an interface to upload the binary file directly. The custom OMA-URI only applies to an MDM service. So I think it is correct to say:
"If you are using Intune, ensure that you select Base64 as the Data type when using Custom OMA-URI functionality, then upload the binary file directly, not encoded as Base64."
It would be helpful if someone could check this with the Intune team. It is certainly what works in practice, and encoding as Base64 certainly does not work. It could be that this is a fault in Intune.

**Comment by mypil** [comment]
@lomayor - Can you please share your insights and review the content of this PR?  Please take note on the suggestions made by @Air-Git.

Thank you.

**Pull request merged**